pkgs/by-name/pa/paperless-ngx/package.nix

```diff
@@ -220,6 +215,7 @@ python.pkgs.buildPythonApplication rec {
       python-ipware
       python-magic
       pyzbar
+      qrcode
```

Comment:
Please use django-allauth.optional-dependencies-mfa

Reply:
I think we want to wait on #374674 for that but I think I get that done by tomorrow.